# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/composite/TextureHost.h
# Commit: 71b6060b6015
# Full Hash: 71b6060b60151e157e94110d28fb51d76bb6415a
# Author: Bob Owen <bobowencode@gmail.com>
# Date: 2023-12-08 04:18:28
# Regressor Bug: 1863914
# File Overlap Count: 1
# Description:
#   Bug 1863914: Use multiple shmem buffers for remote canvas recording. r=aosmond
#   
#   This replaces the use of a single large ring buffer.
#   The buffers are still processed in parallel and are recycled to reduce
#   allocation. Events that do not fit in the default sized buffer have a separate
# ==============================================================================

diff -r bf28b3c066e6 -r 71b6060b6015 gfx/layers/composite/TextureHost.h
--- a/gfx/layers/composite/TextureHost.h	Thu Dec 07 17:10:06 2023 +0000
+++ b/gfx/layers/composite/TextureHost.h	Thu Dec 07 17:27:24 2023 +0000
@@ -21,6 +21,7 @@
 #include "mozilla/ipc/FileDescriptor.h"
 #include "mozilla/layers/CompositorTypes.h"  // for TextureFlags, etc
 #include "mozilla/layers/LayersTypes.h"      // for LayerRenderState, etc
+#include "mozilla/layers/LayersMessages.h"
 #include "mozilla/layers/LayersSurfaces.h"
 #include "mozilla/layers/TextureSourceProvider.h"
 #include "mozilla/mozalloc.h"  // for operator delete